graphh/CGH.py

```diff
@@ -1,45 +1,48 @@
-# coding: utf-8
-import json
+import requests
 import unicodedata
-
+import pandas
+import numpy
+import json
+import warnings
 from graphh import CGHError
-from urllib.request import urlopen
-from urllib.error import HTTPError
 
-class GraphHopper(object):
-    """GraphHopper API class.
 
-    Parameters
-    ----------
-     api_key: str
-        API key to be used for queries
-     premium: bool
-        Whether the account corresponding to this key is a premium account
-        or not
 
+class GraphHopper:
+    """GraphHopper API class.
+
+        Parameters
```

Comment:
Indentation level for the docstring should be that of the beginning of the string (same level as the initial `"""`), cf old version if needed.